modules/ssh/files/allusers.bat

```diff
@@ -0,0 +1,12 @@
+REM This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The scope of this the ssh module should be posix only since windows ssh is handled in another module.  In fact, the scope of this module for right now should be Mojave only.  Therefore remove this file and anything else that does windows or linux stuff.